docs/core/components/content-enrichment-menu.component.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: Content Enrichment Menu Component
+Added: v6.10.0
+Status: Active
+Last reviewed: 2024-06-11
+---
+
+# [Content Enrichment Menu Component](../../../lib/core/src/lib/prediction/components/content-enrichment-menu/content-enrichment-menu.component.ts "Defined in content-enrichment-menu.component.ts")
+
+Allows the user to handle AI predictions by confirming or rejecting changes.
+
+![Content Enrichment Menu Component](../../docassets/images/content-enrichment-menu.png)
+
+## Basic Usage
+
+```html
+<mat-form-field>
+    <mat-label>Form field</mat-label>
+    <adf-content-enrichment-menu matPrefix [prediction]="prediction"></adf-content-enrichment-menu>
+    <input matInput>
+</mat-form-field>
+```
+
+### Properties
+
+| Name       | Type                                                                                                                                     | Default value | Description                       |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------|---------------|-----------------------------------|
+| prediction | [`Prediction`](https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/js-api/src/api/hxi-connector-api/docs/Prediction.md) |               | Prediction for the node property. |
```

Comment:
Is it ok to have a web URL to the Prediction.md file here and not the local file path? Just curious, since I've not seen this being done before

Reply:
changed to relative path